op-node/rollup/derive/channel_out.go

```diff
@@ -14,9 +14,15 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
+var ErrMaxFrameSizeTooSmall = errors.New("maxSize is too small to fit the fixed frame overhead")
 var ErrNotDepositTx = errors.New("first transaction in block is not a deposit tx")
 var ErrTooManyRLPBytes = errors.New("batch would cause RLP bytes to go over limit")
 
+// FrameV0OverHeadSize is the absolute minimum size of a frame.
+// This is the fixed overhead frame size,
+// calculated as follows: 16 + 2 + 4 + 1 = 23 bytes.
```

Comment:
Unclear where those numbers come from but the math checks out

Reply:
comes from the [Frame Format specs](https://github.com/ethereum-optimism/optimism/blob/develop/specs/derivation.md#frame-format)

Reply:
maybe link in this comment.